density/db/db.py

```diff
@@ -189,7 +189,7 @@ def get_oauth_code_for_uni(cursor, uni):
     else:
         # If the code DNE, create a new one and insert into the database.
         new_code = ''.join(random.choice(
-            string.ascii_uppercase + string.digits) for x in xrange(32))
+            string.ascii_uppercase + string.digits) for _x in xrange(32))
```

Comment:
Yeah, I was thinking something along the lines of the first option, as well; I'll get on that, and see what I can eventually do about adding the UNIQUE index + constraint onto the `oauth_data` schema.